taskcat/_dataclasses.py

```diff
@@ -429,7 +446,33 @@ class ProjectConfig(JsonSchemaMixin, allow_additional_props=False):  # type: ign
 
 
 PROPAGATE_KEYS = ["tags", "parameters", "auth"]
-PROPOGATE_ITEMS = ["regions", "s3_bucket", "template", "az_blacklist"]
+PROPOGATE_ITEMS = [
+    "regions",
+    "s3_bucket",
+    "template",
+    "az_blacklist",
+    "s3_regional_buckets",
+]
+
+
+def generate_regional_bucket_name(region_obj: RegionObj, prefix: str = "tcat"):
+    if len(prefix) > 8 or len(prefix) < 1:  # pylint: disable=len-as-condition
+        raise TaskCatException("prefix must be between 1 and 8 characters long")
+    hashed_account_id = hashlib.sha256(
```

Comment:
I'm not sure I see the point of using sha256 here. With just using the raw account id we are guaranteed of uniqueness per AWS account, with taking only the 12 characters of the hash we probably increase the risk of collision, without shortening the bucket name (acc id's are 12 char too).

Or make a uuid5 with the id, the 32 character hex + "taskcat-" + "-<REGION>" should fit nicely into bucket length limits.